src/tool-templates/index-single-in-single-out.tsx

```diff
@@ -73,12 +73,21 @@ const App = () => {
               model.Configuration.doSort ?
                 <Box p={2}> 
                   <InputLabel htmlFor="select">Select a field to order by:</InputLabel>
-                  <Select 
-                    id="select"
+                  <Autocomplete
+                    id="Autocomplete"
                     options={generateData(model.Meta?.fields[0][0].fields)}
+                    renderInput={params => (
+                      <TextField
+                        {...params}
+                        fullWidth
+                        id="ac-meta"
+                        label="Meta Data"
+                        margin="normal"
+                      />
+                    )}
                     onChange={handleFieldSelect}
                     value={model.Configuration.fieldSelect}
-                    placeholderText="Select Column..."
+                    placeholder="Select Column..."
```

Comment:
The `placeholder` prop needs to go on the `TextField`.

Reply:
oh, duh! Fix incoming